buildenv

```diff
@@ -23,3 +40,53 @@ zopen_get_version()
   # Rather than hardcoding the version, obtain the version by running the tool/library
   echo "1.0.0"
 }
+
+# This is called first
+zopen_init()
+{
+  # Check for python
+  if ! type python3 >/dev/null ; then
+    printError "need python3 venv with packages: httplib2 and six"
+  fi
+
+
+  #python3 -m venv curl_python
+  #. ./curl_python/bin/activate
+  #pip3 install impacket
+
+
+  # Ensure path is correct
+  desired_path="zopen/usr/local/zopen/depot_tools/depot_tools-main"
```

Comment:
Each dependency has an environment variable to reflect its path. For depot tools it would be:
$DEPOT_TOOLS_HOME
(you can check the .appenv file to make sure)